installation/quickstart.rst

```diff
@@ -228,80 +227,64 @@ Installation
 Rockstor 4
 ^^^^^^^^^^
 
-Installing Rockstor 4 is particularly quick and straight forward.
+Installing Rockstor 4 is particularly quick and straightforward.
 See the following dedicated doc section for details :ref:`installer_howto`.
 
 Rockstor 3
 ^^^^^^^^^^
 
-Since Rockstor 3 is based on CentOS and uses it's anaconda installer the
-installation looks similar to that of a CentOS or Fedora. Note however that not
-all non-default configurations within this installer are supported by the
-resulting Rockstor 3 install. So it is advised to stick to the defaults where
-possible.
+Since Rockstor 3 is based on CentOS and uses its anaconda installer, the installation looks similar to that of a CentOS or Fedora.
```

Comment:
For another issue/pull request, but we need to have a legacy notice here I think. However we do already have one on the downloads page. But still, it may avoid folks going down a dead-end initially if they are here first.